prov/shm/src/smr_init.c

```diff
@@ -72,6 +72,48 @@ static void smr_resolve_addr(const char *node, const char *service,
 	(*addr)[*addrlen - 1]  = '\0';
 }
 
+static uint64_t smr_calculate_shm_usage(size_t tx_count, size_t rx_count)
```

Comment:
nit: usage to me implies actual use, not need or size.  I would rename usage to one of those. Or just fold this function into the single calling location, which isn't that big anyway.

Reply:
I will rename it to smr_calculate_shm_size.

Reply:
Renamed it to smr_shm_size_per_ep.